bodhi/client/__init__.py

```diff
@@ -164,6 +164,8 @@ def _set_logging_debug(ctx, param, value):
                                                'archived']),
                  help='The state of the release'),
     click.option('--mail-template', help='Name of the email template for this release'),
+    click.option('--not_composed_by_bodhi', is_flag=True, default=False,
```

Comment:
```suggestion
    click.option('--not-composed-by-bodhi', is_flag=True, default=False,
```

Reply:
Also, it'ls possible to get click to name this ```composed_by_bodhi``` and to allow the caller to pass an explicit flag to enable it, like this:

```suggestion
    click.option('--composed-by-bodhi/--not-composed-by-bodhi', is_flag=True, default=True,
```